src/client/dfs/obj.c

```diff
@@ -810,6 +816,9 @@ dfs_ostat(dfs_t *dfs, dfs_obj_t *obj, struct stat *stbuf)
 		D_GOTO(out, rc);
 
 out:
+	if (rc == 0)
+		DFS_OP_STAT_INCR(dfs, DOS_OSTAT);
```

Comment:
same here

Reply:
Ah, I see what you mean. I'll remove the OSTAT metrics.